dot/rpc/modules/state_test.go

```diff
@@ -20,13 +20,15 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/ChainSafe/gossamer/dot/core"
+	"github.com/ChainSafe/chaindb"
```

Comment:
This is an unneeded dependency here, it's only used for `chaindb.ErrKeyNotFound` where you can replace it with a (parent) test local error such as `errTest := errors.New("test error")` and replace all `chaindb.ErrKeyNotFound` occurrences by `errTest` in this file.